src/mon/MonCommands.h

```diff
@@ -114,6 +114,9 @@ COMMAND("pg getmap", "get binary pg map to -o/stdout", "pg", "r", "cli,rest")
 COMMAND("pg send_pg_creates", "trigger pg creates to be issued",\
 	"pg", "rw", "cli,rest")
 COMMAND("pg dump " \
+        "name=poolid,type=CephInt,req=false " \
+        "name=osdid,type=CephOsdName,req=false " \
+	"name=states,type=CephChoices,strings=active|clean|down|replay|spliting|scrubbing|scrubq|degraded|inconsistent|peering|repair|recovery|backfill_wait|incomplete|stale|remapped|deep_scrub|backfill|backfill_toofull|recovery_wait|undersized,n=N,req=false " \
```

Comment:
**scrubq|** is wrong. Could you find a way to match each of these options to the get_pg_state parser below ? You don't need to create a PG in the given state for each of them, that would be quite difficult. But maybe just 

* pg dump replay
* output a message that says : "not pg in replay state" 
* have a test in test.sh that verifies that the output message matches the input argument (that will be proof it has been parsed as expected)